CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+# Contributing to BespokeFit
+
+OpenFF BespokeFit developers follow a [code of conduct]. Please familiarize yourself with the code.
+
+Despite being developed primarily by UK and Australian developers, BespokeFit uses US English in all documentation.
+
+BespokeFit is developed in the open at [GitHub]. We welcome PRs and issues from the general public! Please consider raising an issue there to discuss changes or additions to the code before you commit time to a PR!
+
+A development environment for BespokeFit can be created using [Mamba], which is a faster implementation of the [Conda] package manager:
+
+```shell
+git clone https://github.com/openforcefield/openff-bespokefit.git
+cd openff-bespokefit
+mamba env create --file devtools/conda-envs/test-env.yaml --name bespokefit-dev
+mamba env update --file devtools/conda-envs/docs-env.yaml --name bespokefit-dev
+conda activate bespokefit-dev
+pip install -e .
+```
+
+With this environment, tests can be run locally with [PyTest]:
+
+```shell
+pytest openff/bespokefit/tests
+```
+
+And documentation can be built with [Sphinx] and viewed in a web browser:
+
+```shell
+sphinx-build -j auto docs docs/_build/html
+firefox docs/_build/html/index.html
+```
+
+Finally, BespokeFit uses a number of lints and code formatters to help maintain code quality. These will be run automatically when you submit a PR, but they can be run automatically on each local commit by installing the pre-commit hook:
+
+```shell
+mamba install pre-commit -c conda-forge
+pre-commit run --all-files
+```
+
+Installing this hook locally will only affect this repository and may help avoid frustration associated with the remote branch coming out of sync when multiple commits are pushed in sequence without pulling the automated changes.
```

Comment:
```suggestion
These hooks are installed in isolated virtual environments, separate from the `conda` environment you created for development. Installing `pre-commit` and hooks locally and may help avoid frustration associated with the remote branch coming out of sync when multiple commits are pushed in sequence without pulling the automated changes.
```

Maybe too subtle, but to avoid confusion: `pre-commit` itself is a tool/framework that manages, runs, etc. the hooks; the hooks themselves are each section in the `repos:` header of the config file (i.e. there is one hook for `black`, one hook for `isort`, etc.). We can also mention pre-commit.ci here as further justification for using this framework or if you think it's worth new developers knowing that a bot might commit to their feature branch.

Reply:
Awesome, thanks for that clarification! I did mention that ("These will be run automatically when you submit a PR"), but I'll make it more explicit where it commits.